File: src/main/java/myratpackexamples/webpoll/PollModule.java
package myratpackexamples.webpoll;

import com.google.inject.Binder;
import com.google.inject.Module;

public class PollModule implements Module {
    @Override
    public void configure(Binder binder) {
        binder.bind(CreatePollHandler.class);
    }
}

File: src/main/java/myratpackexamples/webpoll/PollModule.kt
package myratpackexamples.webpoll

import com.google.inject.Binder
import com.google.inject.Module
import myratpackexamples.webpoll.createvote.CreateVoteHandler

class PollModule : Module {
    override fun configure(binder: Binder) {
        binder.bind(CreatePollHandler::class.java)
        binder.bind(RetrievePollHandler::class.java)
        binder.bind(CreateVoteHandler::class.java)
    }
}
